libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts

```diff
@@ -130,6 +140,13 @@ class TigerWorkspaceElementsQuery implements IElementsQuery {
         return {};
     }
 
+    private getGranularity(identifier: string) {
```

Comment:
This is no longer needed as we get the granularity from the backend response itself.

Reply:
ok